src/routes/index.ts

```diff
@@ -252,7 +252,7 @@ router
   .post(trustHistoricalbeneficialOwner.post);
 
 router
-  .route(config.TRUST_ENTRY_URL + config.TRUST_ID + config.TRUST_INDIVIDUAL_BENEFICIAL_OWNER_URL + config.ID + '?')
+  .route(config.TRUST_ENTRY_URL + config.TRUST_ID + config.TRUST_INDIVIDUAL_BENEFICIAL_OWNER_URL + config.BO_ID + '?')
```

Comment:
BO_ID suggests beneficial owner rather than trustee. Maybe TRUSTEE_ID is better if you want to use a different value from ID

Reply:
Yeah good point, I'll create one

Reply:
Thanks

Reply:
done